docs/timelines.md

```diff
@@ -0,0 +1,133 @@
+---
+id: timelines
+title: Timelines introduction
+sidebar_label: Timelines introduction
+---
+
+Timelines allow you to create a precisely timed light show which you can either
+run from the console's internal timer or synchronise to an external timecode
+source. This can be useful for an opening sequence for a corporate
+event, a complex song in a show where the song is pre-recorded or run to 
+a click track or timecode, or a multimedia event run to a playback track.
```

Comment:
We also added a way of using it for one shots, although that isn't the main usage. Not sure if that's worth a mention here or not really.

You can set the fader to automatically play the timeline and pause/stop on zero. Using internal timecode you can use this to create some complicated effects that you can use while busking.